modules/hotp-verification

```diff
@@ -2,12 +2,12 @@ modules-$(CONFIG_HOTPKEY) += hotp-verification
 
 hotp-verification_depends := libusb $(musl_dep)
 
-# v1.4
-hotp-verification_version := b69bb20119d3cea5ec5c13d11b213dd80dfd8334
+# v1.5
+hotp-verification_version := 70c04f51387eee8f777e943ba83b6405764a3cd2
```

Comment:
No regression observed on Nitrokey Pro storage 2 used for HOTP and factory reset.